packages/core/src/render3/component.ts

```diff
@@ -275,12 +278,9 @@ export function createRootContext(
  */
 export function LifecycleHooksFeature(component: any, def: ComponentDef<any>): void {
   const rootTView = readPatchedLView(component)![TVIEW];
```

Comment:
May be we can refactor this code a bit to add an assertion too?
```suggestion
  const lView = readPatchedLView(component)!;
  ngDevMode && assertDefined(lView, 'LView is required');
  const rootTView = lView[TVIEW];
```

Reply:
done